File: src/Elmah.Mvc.Web.Vb/App_Start/ElmahMvc_Start.vb
Imports Elmah.Mvc.Bootstrap

<Assembly: WebActivator.PreApplicationStartMethod(GetType(Elmah.Mvc.Web.App_Start.ElmahMvc_Start), "Start")> 

Namespace Elmah.Mvc.Web.App_Start
    Public Class ElmahMvc_Start
        Public Shared Sub Start()
            Bootstrap.Initialize()
        End Sub
    End Class
End Namespace

File: src/Elmah.Mvc.Web/App_Start/ElmahMvc_Start.cs
[assembly: WebActivator.PreApplicationStartMethod(typeof(Elmah.Mvc.Web.App_Start.ElmahMvc_Start), "Start")]

namespace Elmah.Mvc.Web.App_Start
{
    public class ElmahMvc_Start
    {
        public static void Start()
        {
            Elmah.Mvc.Bootstrap.Initialize();
        }
    }
}